cookbooks/aws-parallelcluster-environment/recipes/init/mount_internal_use_ebs.rb

```diff
@@ -30,6 +30,17 @@
   end
 
 when 'LoginNode'
+  # Mount /opt/parallelcluster/shared over NFS
+  volume "mount #{node['cluster']['shared_dir']}" do
```

Comment:
I agree that is a design choice, but let's discuss about the correctness.
Observations:
1. the separation between shared storage for login and compute nodes have been done to prevent compute nodes from accessing secrets stored in the login nodes shared folder; not the other way around
2. the content of the shared folder is a common content without any secret (mainly config files, changeset, ...)

Reply:
Confirmed with login nodes feature owners. The reason why the shared storage was separated in one folder for login nodes and another one for compute nodes was to prevent compute nodes from reading/writing secrets in lgon nodes shared storage. Login nodes can access compute shared storage in any case by submitting a job that reads/writes it.

Conclusions: it's correct to mount the common shared storage (/opt/parallelcluster/shared) to both compute and login nodes.